packages/ember-runtime/lib/mixins/action_handler.js

```diff
@@ -209,6 +209,20 @@ var ActionHandler = Mixin.create({
                    ') does not have a `send` method', typeof target.send === 'function');
       target.send(...arguments);
     }
+  },
+
+  /**
+   Check if controller has any subscriptions for named event.
+
+   @method hasAction
+   @param {String} actionName The action to trigger
+   @public
+   */
+  hasAction(actionName) {
+    if (this._actions && actionName) {
+      return this._actions.hasOwnProperty(actionName);
```

Comment:
@stefanpenner In this context I think yes. Because ```_action``` is not documented thing, I found it after a little investigate controller implementation. There is reason to believe that this method is not an official - [sounds like internal api](http://stackoverflow.com/questions/26361786/prevent-error-of-missing-handler-in-controller/26364072#26364072). I think it's good to make public documented method ```hasAction```. Firstly it's good point to check listener in child's controllers.